### ISSUE TITLE
Not yet configured

### ISSUE THREAD

**Issue by sebastienb** [arcreative/homebridge-smartthings-routine#3]
The routines are now available but when I try to turn them on or off I get a message back that Routine name is not yet configured. Any clue why that would happen ?

**Comment by arcreative** [comment]
No, not sure... is the message from Homebridge? Maybe something to do with a newer version of Homebridge that this plugin wasn't coded for?
